src/linear_map.rs

```diff
@@ -8,6 +8,7 @@ use Vec;
 /// A fixed capacity map / dictionary that performs lookups via linear search
 ///
 /// Note that as this map doesn't use hashing so most operations are **O(N)** instead of O(1)
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for a `Clone` bound on `N`.